mgmtfn/pslibnet/main.go

```diff
@@ -87,22 +87,23 @@ func main() {
 	err = flagSet.Parse(os.Args[1:])
 	if err != nil {
 		log.Fatalf("Failed to parse command. Error: %s", err)
+		os.Exit(1)
```

Comment:
Just found this: http://www.tldp.org/LDP/abs/html/exitcodes.html

Most of our errors might just fall in exit code 1. We can explore adding our own codes, do you see the exit code being interpreted by the parent scripts/processes?